Server/mods/deathmatch/logic/lua/CLuaManager.cpp

```diff
@@ -93,30 +94,29 @@ bool CLuaManager::RemoveVirtualMachine(CLuaMain* pLuaMain)
 
 void CLuaManager::OnLuaMainOpenVM(CLuaMain* pLuaMain, lua_State* luaVM)
 {
-    MapSet(m_VirtualMachineMap, pLuaMain->GetVirtualMachine(), pLuaMain);
```

Comment:
It seems like MapSet and MapRemove is the standard way to interact with CFastHashMap. What's the purpose of the change here and in OnLuaMainCloseVM?

Reply:
Because `MapSet`, and `MapInsert` as useless. Their only purpose was to provide a better interface for setting/insertion, but since C++11 there are initializer lists, which make those functions totally useless.
That is, pre c++11:
```cpp
m_VirtualMachineMap.insert(std::pair<....>(...))
``` 
Now:
```cpp
m_VirtualMachineMap.insert({...})
```

As for `MapSet`.. I really dont see why it exists in the first place, it is just a simple:
```cpp
m_VirtualMachineMap[key] = value
```

I strongly disagree with the usage of these functions in new code.

Reply:
Isn't `m_VirtualMachineMap[key] = value` cleaner than `m_VirtualMachineMap.insert({key, value})`?

Reply:
It is, thanks!